Packs/CertStream/README.md

```diff
@@ -12,4 +12,48 @@ What does this pack do?
 - Triggers incidents for certificates with high suspicion scores
 - Provides analysts with detailed certificate information to investigate incidents
 
-This pack contains a Certstream integration, parsing scripts, threat intel integrations, and a playbook to generate list of domain names to streamline the end-to-end workflow.
\ No newline at end of file
+This pack contains a Certstream integration, parsing scripts, threat intel integrations, and a playbook to generate list of domain names to streamline the end-to-end workflow.
+
+## Prerequisites
+
+Before using the Certstream Pack and integration, ensure that you have completed the following steps:
```

Comment:
Please add the prerequisites to the integration README to describe the flow before configuring the CertStream pack.